src/ReachSets/discretize.jl

```diff
@@ -1,4 +1,6 @@
 using IntervalMatrices: expm_overapproximation, correction_hull, _expm_remainder
+import MathematicalSystems: discretize
```

Comment:
This was not intended. Or where do we use `MathematicalSystems.discretize`?

Reply:
Ah, you want that `Reachability` *extends* `MathematicalSystems.discretize`. :+1:

Reply:
But then why did you remove the `export discretize` statement?

Reply:
> But then why did you remove the `export discretize` statement?

Because I think it is redundant. For the same reason I removed the `export project`.